docs/src/languages/php/_index.md

```diff
@@ -304,16 +210,197 @@ markdownify=false
+Use grep to get specific values.
+To get the value for `opcache.memory_consumption` you could run something like `php -i | grep opcache.memory_consumption`.
+
+### Customize the PHP settings
+
+There are two ways to customize PHP values for your application.
+The recommended method is to [set variables using the `php` prefix](../../create-apps/app-reference.md#variables).
+For example, to change the PHP memory limit, you'd put the following:
+
+```yaml {location=".platform.app.yaml"}
+variables:
+    php:
+        memory_limit: "256M"
+```
+
+The advantage of this method is that you can use the same files for all your environments and override values on any given environment if needed.
+
+To change the PHP configuration in a specific environment,
+in that environment override the app variables by adding a [PHP-specific environment variable](../../development/variables/_index.md#php-specific-variables).
+
+As an alternative, you can provide a custom `php.ini` file at the [app root](../../create-apps/app-reference.md#root-directory)
+
+```ini {location="php.ini"}
+memory_limit = 256M
+```
+
+For Dedicated, [see the configuration options](../../dedicated/overview/grid.md#php).
+
+### Disable functions for security
+
+A common recommendation for securing PHP installations is disabling built-in functions frequently used in remote attacks.
+By default, Platform.sh doesn't disable any functions.
+
+If you're sure a function isn't needed in your app, you can disable it.
+
+For example, to disable `pcntl_exec` and `pcntl_fork`:
+
+```yaml {location=".platform.app.yaml"}
+variables:
+    php:
+        disable_functions: "pcntl_exec,pcntl_fork"
+```
+
+Common functions to disable include:
+
+| Name | Description |
+|------|-------------|
+| `create_function` | This function has been replaced by anonymous functions and shouldn't be used anymore.|
+| `exec`, `passthru`, `shell_exec`, `system`, `proc_open`, `popen` | These functions allow a PHP script to run a bash shell command. Rarely used by web applications except for build scripts that might need them. |
+| `pcntl_*` | The `pcntl_*` functions are responsible for process management. Most of them cause a fatal error if used within a web request. Cron tasks or workers may need them. Most are usually safe to disable. |
+| `curl_exec`, `curl_multi_exec` | These functions allow a PHP script to make arbitrary HTTP requests. If you're using HTTP libraries such as Guzzle, don't disable them. |
+| `show_source` | This function shows a syntax highlighted version of a named PHP source file. Rarely useful outside of development. |
+
+## Alternate start commands
+
+For further control over your app or to run a custom PHP script, a web server or a thread-based worker process, set a `start` command.
+See more on [web commands reference](../../create-apps/app-reference.md#web-commands).
+
+PHP-CLI (command line interface) is the mode used for alternate start commands which differs from PHP-CGI (common gateway interface).
```

Comment:
:lady_beetle: This is not the appropriate way to define abbreviations. It's also unclear how a mode affects what can be done here.